client/src/components/ReviewTile.js

```diff
@@ -1,23 +1,96 @@
-import React from "react";
+import React, { useState } from "react";
+import EditReviewForm from "./EditReviewForm";
+import ErrorList from "./layout/ErrorList";
 
-const ReviewTile = ({ rating, content, onDelete, id, currentUser, userId, userName }) => {
+const ReviewTile = ({ rating, content, onDelete, id, currentUser, userId, memeId, setMeme, meme }) => {
+  const [editForm, setEditForm] = useState(false);
+  const [errors, setErrors] = useState([]);
+
+  const handleEditButton = (event) => {
+    event.preventDefault();
+    if (editForm) {
+      setEditForm(false);
+    } else {
+      setEditForm(true)
+    }
+  };
 
   const handleDeleteButton = (event) => {
-    event.preventDefault()
-    onDelete(id)
+    event.preventDefault();
+    onDelete(id);
+  };
+
+  const editReview = async (reviewId, reviewData) => {
+    try {
+      const response = await fetch(`/api/v1/memes/${memeId}/reviews/${reviewId}`, {
```

Comment:
a review belongs to one meme. As such, on the backend, we dont need a memeId to find this review's related meme.

as such, update the path to be `"/api/v1/reviews/${reviewId}"` this will likely change up a few things